release-calendar/src/client/components/Calendar.tsx

```diff
@@ -88,7 +98,7 @@ export class Calendar extends React.Component<CalendarProps, CalendarState> {
           right: 'dayGridMonth,timeGridWeek,listWeek',
         }}
         plugins={[dayGridPlugin, timeGridPlugin]}
-        eventSources={displayEvents}
+        eventSources={[{events: displayEvents, textColor: 'white'}]}
```

Comment:
Because of this, where I put the `ReleaseEventInput[]` into a `EventSourceInput`, there isn't a need for `EventSourceInput` to be imported in any of my file.

Reply:
SG